### DISCUSSION TITLE
Can't run certain commands like /recipie

### DISCUSSION THREAD

**Discussion by Pabsb** [EssentialsX/Essentials#4142] [Q&A] [answered]
Get an error message with "This Feature is not supported on the current server version."

/ess version shows:
`Server version: 1.16.5-R0.1-SNAPSHOT 2991-Spigot-018b9a0-f3f3094 (MC: 1.16.5)
EssentialsX version: 2.19.0-dev+113-0d640d1
LuckPerms version: 5.3.3
Vault version: 1.7.3-b131
EssentialsXSpawn version: 2.19.0-dev+113-0d640d1
EssentialsXChat version: 2.19.0-dev+113-0d640d1
You are running a server with limited API functionality. EssentialsX will still work, but certain features may be disabled.
Fetching version information...
You're running the latest EssentialsX dev build!`

**Comment by triagonal** [comment, marked as answer]
The use of /recipe requires Paper (or its derivatives) to be used as the server software. Paper provides an extra event that is necessary to prevent a duplication glitch in /recipe. Download the latest version of Paper at https://papermc.io/downloads.
